### ISSUE TITLE
MSID: 89679 Version: 1 DOI: 2023.07.26.550653

### ISSUE THREAD

**Issue by nlisgo** [elifesciences/enhanced-preprints-import#922]
[PLACE MANUSCRIPT AND EDITOR DETAILS HERE WHEN AVAILABLE] See step 3

[PLACE PDF URL HERE WHEN AVAILABLE] See step 7

## Step 1. Inform bioRxiv

Who can help: @QueenKraken, @nlisgo, @scottaubrey

- [ ] Manuscript is in data hub index (https://data-hub-api.elifesciences.org/enhanced-preprints/docmaps/v1/by-publisher/elife/get-by-manuscript-id?manuscript_id=89679)

or (only one should be ticked. remove other from description.)

- [ ] Confirmed with Ted at bioRxiv that MECA archive is available

Send the following email to Ted and wait for his reply.

```
Hi Ted,

Please can you prepare the preprint MECA for 10.1101/2023.07.26.550653

Thanks
```

## Step 2. Create preview of manuscript

Who can help: @fred-atherden, @nlisgo, @scottaubrey

- [ ] Manuscript is available for preview (https://prod--epp.elifesciences.org/preview/89679/v1)
    - [ ] Trigger import for staging: https://prod--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://prod-elife-epp-data/data
    - [ ] Trigger import for staging: https://staging--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://staging-elife-epp-data/data
    - [ ] Pull request created to [enhanced-preprint-data](https://github.com/elifesciences/enhanced-preprints-data/pulls)
    - [ ] Post link to pull request below

Pull request: [PLACE LINK TO PULL REQUEST HERE]

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-manuscript

## Step 3: Awaiting public reviews and QC

Who can help: Production team

- [ ] QC passed or QC not needed

Who can help: Editorial team

- [ ] Public reviews available on sciety (https://sciety.org/articles/activity/10.1101/2023.07.26.550653)
- [ ] Manuscript and editor details at the top of this issue (Supplied by Editorial team)
- [ ] Reviews returned by EPP - https://prod--epp.elifesciences.org/api/reviewed-preprints/89679/v1/reviews

<details>
<summary>Example</summary>

```
"msas": "Genetics and Genomics", "Neuroscience"
"msid": "84628"
"version": "1"
"preprintDoi": "10.1101/2022.10.28.514241"
"articleType": "Reviewed Preprint"
"status": "Published from the original preprint after peer review and assessment by eLife."

"Reviewed Preprint posted": "2023-01-02"
"Sent for peer review": "2022-10-28"
"Posted to bioRxiv": "2022-11-21" (link: "Go to bioRxiv": "https://www.biorxiv.org/content/10.1101/2022.10.28.514241v1")

Editors:
Reviewing Editor
Michael B Eisen
University of California, Berkeley, United States
Senior Editor
Michael B Eisen
University of California, Berkeley, United States
```

</details>

## Step 4: Modify manuscripts.json (no PDF)

- [ ] Reviewed preprint is published to EPP (https://elifesciences.org/reviewed-preprints/89679)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below
- [ ] Request that a doi be registered
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

Pull request: [PLACE LINK TO PULL REQUEST HERE]
[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

<details>
<summary>Instructions to modify manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/publish-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/334/files

Once the pull request is merged in it should be available a few minutes later.

</details>

### Request that a doi

Post the following in [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77):

```
@Fred can you register a doi for https://elifesciences.org/reviewed-preprints/89679
```

## Step 5: Awaiting search reindex

- [ ] Reviewed preprint is avaliable on journal homepage (https://elifesciences.org)

The search reindex is triggered once an hour. We need the reviewed preprint to be indexed as the search application serves the journal homepage.

<details>
<summary>Additional info</summary>

If needed, the jenkins pipeline to reindex search can be triggered sooner.

https://alfred.elifesciences.org/job/process/job/process-reindex-reviewed-preprints/

</details>

## Step 6: Published! Request PDF generation

- [ ] Post the link to the PDF url at the top of the issue
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

Post the following to the [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) on slack:

```
@Ryan Dix-Peek please can you generate a PDF for https://elifesciences.org/reviewed-preprints/89679
```

## Step 7: Introduce PDF to data folder and git repo

- [ ] PDF is avaliable at https://github.com/elifesciences/enhanced-preprints-data/raw/master/data/89679/v1/89679-v1.pdf

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-pdf

## Step 8: Add PDF url to manuscripts.json

- [ ] Reviewed preprint PDF is available for download (https://prod--epp.elifesciences.org/reviewed-preprints/89679)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below

[PLACE LINK TO PULL REQUEST HERE]

<details>
<summary>Instructions to add PDF url to manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/add-pdf-url-to-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/397/files

Once the pull request is merged in it should be available a few minutes later.

</details>

## Step 9: Done!

- [ ] Kettle is on!

**Comment by fred-atherden** [comment]
The fonts don't match between equation images and the HTML (because the glyphs are different in the PDF and the equations are images taken from the PDF):


![Image](https://github.com/elifesciences/enhanced-preprints-import/assets/43879983/fdb72e51-322e-4d62-932c-9dbddfe879e6)


Short of getting bioRxiv (or Exeter) to capture the specific glyphs also as images, there's nothing we can do here, and since it's in the original on bioRxiv I think it's fine. 


However there are issues with table and figure citations throughout. Here are some examples:


![Image](https://github.com/elifesciences/enhanced-preprints-import/assets/43879983/af3a9072-5225-4c35-a20f-6ca46997932b)




![Image](https://github.com/elifesciences/enhanced-preprints-import/assets/43879983/9196131f-f033-40b5-9433-3b3da1067a64)



![Image](https://github.com/elifesciences/enhanced-preprints-import/assets/43879983/e5830e84-61bf-4037-aaf1-7ef1092d8da7)



And also, as in the last screenshot above, the reference citation style is incorrect throughout (should be in brackets). 

@JGilbert-eLife do you know if the reference citation style is an issue with our overleaf template? Or is this simply a mistake from the authors? 


@acollings do you think it's worth asking the authors about these issues to do with citations (or at least just the broken figure/table ones)?

**Comment by JGilbert-eLife** [comment]
Not sure how to classify the reference thing - the template allows both styles, but clearly the authors have picked the wrong one here.

![Screenshot 2023-08-24 at 12 09 30](https://github.com/elifesciences/enhanced-preprints-import/assets/1785305/7e28fc33-7acc-42a6-8702-86bce63172ec)

**Comment by fred-atherden** [comment]
Good to know - thanks James.

**Comment by acollings** [comment]
I'll ask the authors: thanks!

**Comment by fred-atherden** [comment]
@acollings have we heard back from the authors of this one at all?

**Comment by acollings** [comment]
Hi @fred-atherden thanks for checking. The authors seem very confused, I'm afraid. They emailed a new PDF to me, but didn't update the preprint. I explained again they need to update the preprint 11 days ago, but haven't heard more. I'm not sure what to do next: should we proceed and publish despite the issues? Or I can send you the PDF they sent, if it helps?

**Comment by fred-atherden** [comment]
Thanks @acollings. Would it be OK to try and explain what we need them to do again and if that fails we'll explore other options? 

I think it'd be better to get Exeter to typeset this one than publish as is, but obviously it'd be preferable of we use a bioRxiv version.

**Comment by acollings** [comment]
Yep, will do now @fred-atherden

**Comment by acollings** [comment]
Hi @fred-atherden, the new preprint is up! https://www.biorxiv.org/content/10.1101/2023.07.26.550653v2

**Comment by fred-atherden** [comment]
Nice one - thanks Andy! I'll keep an eye out for the MECA package appearing.

**Comment by fred-atherden** [comment]
MECA package available at `s3://transfers-elife/biorxiv_Current_Content/September_2023/07_Sep_23_Batch_1727/c950c731-6d9e-1014-b9d8-bf71361d0298.meca`

**Comment by fred-atherden** [comment]
First author is noted as corresponding and has another authors email as a result. I think this is a mistake on the authors side, and given the delays already and the issues with getting this fixed I think we should live with it.
